src/ddprof_worker.cc

```diff
@@ -245,10 +246,8 @@ static DDRes ddprof_unwind_sample(DDProfContext &ctx, perf_event_sample *sample,
   if (watcher->config == PERF_COUNT_SW_TASK_CLOCK)
     ddprof_stats_add(STATS_TARGET_CPU_USAGE, sample->period, NULL);
 
-  // Attempt to fully unwind if the watcher has a callgraph type
-  DDRes res = {};
-  if (AnyCallgraph(watcher->output_mode))
-    res = unwindstate__unwind(us);
+  // Attempt to fully unwind for this watcher
+  DDRes res = unwindstate__unwind(us);
```

Comment:
Metrics are not yet a thing and need more work. I'm just removing the branch for now.

Reply:
Metrics belong in a different field.
They don't change the value computation, they change the way we report the data.